accounts-db/src/accounts_index.rs

```diff
@@ -2241,10 +2243,10 @@ pub mod tests {
         let key = solana_pubkey::new_rand();
 
         let mut config = ACCOUNTS_INDEX_CONFIG_FOR_TESTING;
-        config.index_limit_mb = if use_disk {
-            IndexLimitMb::Minimal
+        config.index_limit = if use_disk {
+            IndexLimit::Minimal
         } else {
-            IndexLimitMb::InMemOnly // in-mem only
+            IndexLimit::InMemOnly // in-mem only
```

Comment:
While we're here, seems like we can remove the unnecessary comment.
```suggestion
            IndexLimit::InMemOnly
```